config/initializers/new_framework_defaults_7_1.rb

```diff
@@ -0,0 +1,283 @@
+# Be sure to restart your server when you modify this file.
```

Comment:
This might be a good time to delete `config/initializers/new_framework_defaults_7_0.rb` unless you still want to hang on to it.